src/render-instruction.ts

```diff
@@ -65,7 +65,7 @@ class InstructionRenderer {
       typeMapper
     )
     this.programIdPubkey = `new ${SOLANA_WEB3_EXPORT_NAME}.PublicKey('${this.programId}')`
-    this.defaultOptionalAccounts = ix.defaultOptionalAccounts ?? false
+    this.defaultOptionalAccounts = !ix.legacyOptionalAccountsStrategy
```

Comment:
Does it make sense to check for `ix.defaultOptionalAccounts` to maintain backwards compatibility?

Reply:
As discussed in private, this is not relevant since the breaking change occurs when `defaultOptionalAccounts` is not set.